packages/builders/src/messages/embed/Assertions.ts

```diff
@@ -25,7 +25,12 @@ export const authorNamePredicate = fieldNamePredicate.nullable();
 
 export const urlPredicate = z.string().url().nullish();
 
-export const colorPredicate = z.number().gte(0).lte(0xffffff).nullable();
+export const colorPredicate = z
+	.number()
+	.gte(0)
+	.lte(0xffffff)
+	.nullable()
+	.or(z.tuple([z.number(), z.number(), z.number()]));
```

Comment:
Perhaps you should add more validation to the RGB values too? (e.g. int, min, max, etc.)